FrontEnd/src/__tests__/App.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { act } from "react-dom/test-utils";
+import App from '../App';
+
+
+let div = null;
+
+beforeEach(() => {
+  // setup a DOM element as a render target
+  div = document.createElement("div");
+  document.body.appendChild(div);
+});
+
+
+it('renders without crashing', () => {
+  act(() => {
+    ReactDOM.render(<App />, div);
+  })
```

Comment:
This test results in errors even if it passes that send mixed messages. I would ignore it until you can create is safely.